src/test/java/ui/ConfigProperties.java

```diff
@@ -0,0 +1,31 @@
+package ui;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    protected static Properties properties;
+    protected static FileInputStream fileInputStream;
+
+    static {
+        try {
+            fileInputStream = new FileInputStream("src/test/resources/config.properties");
+            properties = new Properties();
+            properties.load(fileInputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
In case of IOException properties won't be read. What's the point of swallowing the exception and letting the application run?
Would it be better to throw this exception wrapped in some runtime one?